wp-content/themes/missilethreat/assets/_scss/components/_image.scss

```diff
@@ -8,6 +8,10 @@
   }
 }
 
+p img {
```

Comment:
Are there cases where an image tag is inside of a paragraph?

Reply:
Yes, on some older posts images aren't in a `figure` tag, they're directly in the `p` tag. [Example](https://localhost:3000/defsys/dsp/)

Reply:
Got it. Good to know! Sorry, I see now you left that in your description... my bad!